app/src/main/java/org/wikipedia/analytics/ABTestExploreVsHomeFunnel.java

```diff
@@ -0,0 +1,20 @@
+package org.wikipedia.analytics;
+
+public final class ABTestExploreVsHomeFunnel extends ABTestFunnel {
+    public ABTestExploreVsHomeFunnel() {
+        super("exploreVsHome", ABTestFunnel.GROUP_SIZE_2);
+    }
+
+    public boolean shouldSeeHome() {
+        boolean enrolled = isEnrolled();
+        boolean showHome = getABTestGroup() == ABTestFunnel.GROUP_1;
```

Comment:
Should we have `ReleaseUtil.isPreBetaRelease()` here for testing purposes?

Reply:
We have developer preferences for testing purposes now. (set the developer preference to an even number for the A group, or an odd number for the B group.)

Reply:
I see. I set it as `123` and that's why it does not show for me 😅